specification/monitor/resource-manager/microsoft.insights/2017-05-01-preview/examples/getDiagnosticSetting.json

```diff
@@ -0,0 +1,43 @@
+{
+  "parameters" : {
+    "resourceUri" : "/subscriptions/1a66ce04-b633-4a0b-b2bc-a912ec8986a6/resourcegroups/viruela1/providers/microsoft.logic/workflows/viruela6",
+    "name": "mysetting",
+    "api-version" : "2017-05-01-preview"
+  },
+  "responses" : {
+    "200" : {
+      "headers" : {},
+      "body" : {
+        "id" : "/subscriptions/1a66ce04-b633-4a0b-b2bc-a912ec8986a6/resourcegroups/viruela1/providers/microsoft.logic/workflows/viruela6/diagnosticSettings/service",
+        "type" : "",
+        "name" : "mysetting",
+        "location" : "",
+        "tags" : null,
+        "properties" : {
+          "storageAccountId" : "/subscriptions/df602c9c-7aa0-407d-a6fb-eb20c8bd1192/resourceGroups/apptest/providers/Microsoft.Storage/storageAccounts/appteststorage1",
+          "serviceBusRuleId" : "/subscriptions/b67f7fec-69fc-4974-9099-a26bd6ffeda3/resourceGroups/andy1101/providers/Microsoft.EventHub/namespaces/andy1101/authorizationrules/RootManageSharedAccessKey",
```

Comment:
The response model `DiagnosticSettings` doesn't have `ServiceBusRuleId` property and no additional properties are accepted.